purestorage/purestorage.py

```diff
@@ -2001,7 +2001,7 @@ def test_snmp_manager(self, manager):
     # Note: These methods only work with REST API 1.2 and later
     #
 
-    def connect_array(self, address, connection_key, connection_type, **kwargs):
+    def connect_array(self, management_address, connection_key, connection_type, **kwargs):
         """Connect this array with another one.
 
         :param address: IP address or DNS name of other array.
```

Comment:
Just the comments on line 2007 and 2008, they get used by sphinx and end up being used for this documentation http://pythonhosted.org/purestorage/api.html#purestorage.FlashArray.connect_array

Not sure its documented, but if you cd into the docs dir and use "make" (I usually use "make singlehtml" when checking code changes) it will generate them and you can see the output in ./docs/_build/index.html